### DISCUSSION TITLE
Error loading report with more than 50 records

### DISCUSSION THREAD

**Discussion by VitorMavel** [dompdf/dompdf#2522] [General]
I'm using version 1.0.2 of dompdf, when I put up to 50 records +/- I normally open the pdf but when I need more than that, although the download gets corrupted:
Below are two images, the first with only about 50 records and the other with the error:

![image](https://user-images.githubusercontent.com/88976799/129482859-050716b6-1c0e-4c05-8da3-5bb026e494ab.png)
![image](https://user-images.githubusercontent.com/88976799/129482896-0fe6440f-47ee-4af0-9a31-6fccf054d419.png)

**Comment by bsweeney** [comment]
Can you post the corrupted PDF? It's likely the case that the format of the output has already been set, but that the server threw an error during processing (maybe an execution timeout?). If that's the case, viewing the PDF in a text editor may reveal useful info.
